utils/mountd/cache.c

```diff
@@ -881,14 +881,19 @@ static void write_secinfo(char **bp, int *blen, struct exportent *ep, int flag_m
 
 }
 
+/* The ttl parameter represents the amount of time the cache entry will be
```

Comment:
Nit: Block comments should start with a blank line: `/*\n`

Reply:
I was trying to match the style in the existing code, but looking closer, it doesn't really seem to use a consistent style.